components/woocommerce/woocommerce.app.mjs

```diff
@@ -196,9 +205,15 @@ export default {
     async listCategories(page) {
       return this.listResources(`products/categories?page=${page}`);
     },
+    async getOrder(id) {
```

Comment:
If you add async options for `orderId`, you'll need a `listOrders` method to retrieve the orders.
```suggestion
    async listOrders(page) {
      return this.listResources(`orders?page=${page}`);
    },
    async getOrder(id) {
```